playbook/app/pb_kits/playbook/pb_form/formHelper.js

```diff
@@ -0,0 +1,39 @@
+const formHelper = () => {
+    const loadingForm = document.querySelector(".pb_form_loading")
+    if (loadingForm) {
+        loadingForm.addEventListener("submit", function(event) {
+            const submitButton = event['submitter'];
+            const cancelButton = event['target'].querySelector('button[type="reset"]');
+
+            if (submitButton) {
+                let currentClass = submitButton.className;
+                let newClass = currentClass.replace("_enabled", "_disabled_loading");
+
+                let cancelClass = cancelButton ? cancelButton.className : "";
+                let newCancelClass = cancelClass.replace("_enabled", "_disabled");
+
+                submitButton.disabled = true;
+                submitButton.className = newClass;
+
+                if (cancelButton) {
+                    cancelButton.disabled = true;
+                    cancelButton.className = newCancelClass;
+                }
+
+                setTimeout(function() {
+                    submitButton.disabled = false;
+                    submitButton.className = currentClass;
+
+                    if (cancelButton) {
+                        cancelButton.disabled = false;
+                        cancelButton.className = cancelClass;
+                    }
+                }, 5000);
```

Comment:
Great work on this PR, @skduncan !

One thing we realized after looking through your code, is that this piece (the 5000ms) will need to be in the docs code only. It looks like you set the timeout for one of the docs examples, but excluded it from the other. We can't ship this timeout in the actual formHelper, because then it will be put on every instance of the form that brings in the formHelper. 

Let's chat on Monday morning, if you have some time! And we can figure this out!